recipe/meta.yaml

```diff
@@ -9,7 +9,9 @@ source:
   url:
     - http://zlib.net/zlib-{{ version }}.tar.gz
     - https://gnupg.org/ftp/gcrypt/zlib/zlib-{{ version }}.tar.gz
-  sha256: b3a24de97a8fdbc835b9833169501030b8977031bcb54b3b3ac13740f846ab30
+  sha256: ff0ba4c292013dbc27530b3a81e1f9a813cd39de01ca5e0f8bf355702efa593e
+  patches:
+    - CVE-2023-45853.patch
```

Comment:
Can you add a comment with the PR (https://github.com/madler/zlib/pull/843) and some text reminding us to drop it in the new release?